.gitlab-ci.yml

```diff
@@ -88,6 +88,20 @@ after_script:
 
     - set +e
 
+.dune-template: &dune-template
+  stage: build
+  artifacts:
+    name: "$CI_JOB_NAME"
+    paths:
+      - _build/
+    expire_in: 1 week
+  script:
+    - set -e
+    - echo 'start:coq.dune'
+    - make -f Makefile.dune world
+    - echo 'end:coq.dune'
+    - set +e
```

Comment:
Just keep the make line, the echos aren't really useful.

Reply:
Why are the markers a mistake? I don't see why they bother us. That being said, yeah, I don't mind removing them either way.